doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -913,6 +923,20 @@ Note that if a custom payload is used for the notification (the field `payload`
 corresponding subscription), then a value of `custom` is used for the `Ngsiv2-AttrsFormat` header
 in the notification.
 
+## Special metadata in notifications
+
+The following metadata are not included in notifications by default, the `metadata` field (within
+`notification` field) in subscription is used to do so.
+
+* `previousValue`: the value and type of this metadata is the one of the attribute to which it
```

Comment:
This only can happen in a actionType = append case, which is out of the scope of this PR. We can defer the decission until #1494 gets implemented. What do you think?